mobile/src/main/java/org/openhab/habdroid/core/message/MessageHandler.java

```diff
@@ -19,8 +18,6 @@
     public static final int TYPE_TOAST = 3;
 
     public static final int LOGLEVEL_DEBUG = 0;
-    public static final int LOGLEVEL_REMOTE = 1;
```

Comment:
They are used in current master, so they should also be used here. In case you have no remote connection, you won't see "Connecting to local URL".